site/source/user-manual/service-guides/matrix/synapse.rst

```diff
@@ -0,0 +1,46 @@
+.. _synapse:
+
+=======
+Synapse
+=======
+
+.. contents::
+  :depth: 2 
+  :local:
+
+
+.. _synapse-admin:
+
+-------------------
+Creating a new user
+-------------------
+
+These instructions will guide you through creating your first Matrix user on your Synapse service via the admin interface.
+
+#. Visit the instructions page by clicking `Services > Synapse > Instructions` and following along.
+
+#. As you come to **Step 2: Creating Your First Account**, continue below for instructions augmented with screenshots:
+
+  **Substep 1**: In your Synapse dashboard, click "Launch UI"
```

Comment:
Maybe check how 'substeps' are done elsewhere.  I think I normally turn them into dashed lists, but it's possible we have used 1a., 1b., etc type listing